docs/code_of_conduct.md

```diff
@@ -20,13 +20,27 @@ Private or public harassment of any kind will not be tolerated. Since harassment
 
 Additionally, spam and other content which disrupts or prevents LDK contributors from working on LDK is not acceptable.
 
+Deadlock can arise about the application fo the code of conduct to a concrete issue, where a deadlock is defined as lack of unanimity between the Code of Conduct team on the decision to be taken.
+
+In that case, 2 additional members should be selected by consensus of the code of conduct team from the community of active LDK contributors to join the team in its decision-making process to adjudicate
+the issue. The decision of the majority of 5 members is binding.
```

Comment:
IMO this is unnecessary complexity, a benefit of having an odd number of people is that there will never be a deadlock and the majority decision can always be taken.